commandapi-nms/commandapi-1.19-common/src/main/java/dev/jorel/commandapi/nms/NMS_1_19_Common.java

```diff
@@ -577,6 +536,111 @@ public final ParticleData<?> getParticle(CommandContext<CommandSourceStack> cmdC
 		return new ParticleData<Void>(particle, null);
 	}
 
+	/**
```

Comment:
JavaDocs with no Javadocs! You can't have JavaDocs with no description for the parameters!

In this case however, we don't need JavaDocs here because this is a `private` method.